clients/drcachesim/tracer/raw2trace.h

```diff
@@ -931,8 +931,13 @@ class raw2trace_t {
     /**
      * The trace_entry_t buffer returned by get_write_buffer() is assumed to be at least
      * #WRITE_BUFFER_SIZE large.
+     *
+     * Some of the AArch64 SVE scatter/gather instructions have a lot of memref entries.
+     * For example ld4b loads 4 registers with byte sized elements, so that is
+     *     (vl_bits / 8) * 4
+     * entries. With a 512-bit vector length that is (512 / 4 * 8) = 256 memref entries.
```

Comment:
Do we want this to be large enough to fit one instr with its memrefs and that's why we've set it to 256 + 1 rounded up to 260? Maybe add to comment that this should be worth the largest instr + its memrefs.

Reply:
Done.